internal/namespaces/instance/v1/testdata/test-create-server-errors-error:-invalid-image-uuid.stderr.golden

```diff
@@ -1 +1,3 @@
+WARNING: 2019/12/09 16:04:07 cannot get image 7a892c1a-bbdc-491f-9974-4008e3708664: scaleway-sdk-go: http error 404 Not Found: "7a892c1a-bbdc-491f-9974-4008e3708664" not found
```

Comment:
Missing cassette

Reply:
Error is ok

This image does not exist